scripts/sonic_py.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+
+"""
+    Sonic_py is the library that provides APIs to access database.
+"""
+
+import redis
+
+
+REDIS_UNIX_SOCKET = "/var/run/redis/redis.sock"
+CNT_DB = 2
+COUNTER_PORT_NAME_MAP = "COUNTERS_PORT_NAME_MAP"
+
+class sonic_py(object):
```

Comment:
Thanks, Pavel. I was unaware that library existed. if it works, we should use it and not reinvent the wheel. Sihui, please take a look.